README.md

```diff
@@ -132,6 +132,8 @@ Of course there's nothing stopping you from using if-statements to detect whethe
 ## Installation
 Add this line to your application's Gemfile:
 
+### Install the gem
```

Comment:
Because I wanted to add the information about the `application.rb` I tried to break this up into more sections. Not sure what the thoughts are on it.

Reply:
Now that I've reverted 538fa37fba0562b134dc5dc07cc0fd9695ce51cb, would it make sense to revert this change as well?